docs/daytona_server_logs.md

```diff
@@ -10,6 +10,7 @@ daytona server logs [flags]
 
 ```
   -f, --follow   Follow logs
+  --file, Read logs from local server log file
```

Comment:
This file should not be modified manually. Please run `./hack/generate-cli-docs.sh`